Makefile

```diff
@@ -121,19 +121,31 @@ vet: ## Run go vet against code.
 	go vet ./...
 
 .PHONY: test-converged
-test-converged: manifests generate fmt vet envtest ## Run converged mode tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" DEPLOYMENT_TYPE=converged go test ./... -coverprofile cover.out
+test-converged: manifests generate fmt vet envtest ## Run converged mode tests with network type sdn.
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" DEPLOYMENT_TYPE=converged NETWORK_TYPE=sdn go test ./... -coverprofile cover.out
 
 .PHONY: test-provider
-test-provider: manifests generate fmt vet envtest ## Run provider mode tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" DEPLOYMENT_TYPE=provider go test ./... -coverprofile cover.out
+test-provider: manifests generate fmt vet envtest ## Run provider mode tests with network type sdn.
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" DEPLOYMENT_TYPE=provider NETWORK_TYPE=sdn go test ./... -coverprofile cover.out
 
 .PHONY: test-consumer
-test-consumer: manifests generate fmt vet envtest ## Run consumer mode tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" DEPLOYMENT_TYPE=consumer go test ./... -coverprofile cover.out
+test-consumer: manifests generate fmt vet envtest ## Run consumer mode tests with network type sdn.
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" DEPLOYMENT_TYPE=consumer NETWORK_TYPE=sdn go test ./... -coverprofile cover.out
+
+.PHONY: test-converged-ovn
+test-converged-ovn: manifests generate fmt vet envtest ## Run converged mode tests with network type ovn.
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" DEPLOYMENT_TYPE=converged NETWORK_TYPE=ovn go test ./... -coverprofile cover.out
+
+.PHONY: test-provider-ovn
+test-provider-ovn: manifests generate fmt vet envtest ## Run provider mode tests with network type ovn.
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" DEPLOYMENT_TYPE=provider NETWORK_TYPE=ovn go test ./... -coverprofile cover.out
+
+.PHONY: test-consumer-ovn
+test-consumer-ovn: manifests generate fmt vet envtest ## Run consumer mode tests with network type ovn.
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" DEPLOYMENT_TYPE=consumer NETWORK_TYPE=ovn go test ./... -coverprofile cover.out
 
 .PHONY: test
-test: test-converged test-provider test-consumer ## Run tests.
+test: test-converged test-provider test-consumer test-converged-ovn test-provider-ovn test-consumer-ovn ## Run tests.
```

Comment:
It is meaningless to run the entire test suite with both OVN and SDN variants and is a waste of time.
The OVN vs SDN decision is only affecting a very small portion of our codebase.